monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -180,7 +180,19 @@ class ConfigurePageComponent extends AuthComponent {
       if (techType[1].properties.hasOwnProperty(technique)) {
         let tempMatrix = this.state.attackConfig;
         tempMatrix[techType[0]].properties[technique].value = value;
-        this.setState({attackConfig: tempMatrix});
+
+        if (technique == 'T1136') {
+          let newConfig = this.state.configuration;
+          if (value && !newConfig['monkey']['general']['post_breach_actions'].includes('BackdoorUser')) {
+              newConfig['monkey']['general']['post_breach_actions'].push('BackdoorUser');
+            }
+          else if (!value && newConfig['monkey']['general']['post_breach_actions'].includes('BackdoorUser')) {
+              let toRemoveIndex = newConfig['monkey']['general']['post_breach_actions'].indexOf('BackdoorUser');
+              newConfig['monkey']['general']['post_breach_actions'].splice(toRemoveIndex, 1);
+            }
+          this.setState({attackConfig: tempMatrix, configuration: newConfig});
+          this.configSubmit();
+      }
```

Comment:
What does this code achieve? You try to enable/disable "Backdoor user" PBA as you enable/disable T1136 on the matrix? 
Try to remove this code and it should still happen automatically, the same way it happens with other attack techniques. Test it, OFC. If it doesn't happen automatically leave a comment and I'll look into why.

Reply:
Tested again; doesn't happen automatically

Reply:
I pushed a bugfix, there was a typo on configuration schema.

Reply:
A possible bug now is that you might switch tabs to monkey tab before configuration updates from backend. If this happens open up an issue, but it's beyond the scope of this PR

Reply:
Tried that a bunch of times; isn't happening for me